controllers/front/PostComment.php

```diff
@@ -134,7 +134,7 @@ private function validateComment(ProductComment $productComment)
         if (empty($productComment->getTitle())) {
             $errors[] = $this->trans('Title cannot be empty', [], 'Modules.Productcomments.Shop');
         } elseif (strlen($productComment->getTitle()) > ProductComment::TITLE_MAX_LENGTH) {
-            $errors[] = $this->trans('Title cannot be more than %s characters', [ProductComment::TITLE_MAX_LENGTH],'Modules.Productcomments.Shop');
+            $errors[] = $this->trans('Title cannot be more than %s characters', [ProductComment::TITLE_MAX_LENGTH], 'Modules.Productcomments.Shop');
```

Comment:
Are we okay that this string regards the front office?

Reply:
Yes it is returned in the ajax call